api/src/main/java/com/codeforcommunity/rest/subrouter/BlocksRouter.java

```diff
@@ -63,6 +66,16 @@ private void registerGetReserved(Router router) {
     reserveRoute.handler(this::handleGetReserved);
   }
 
+  private void registerGetReservedAdmin(Router router) {
+    Route reservedAdminRoute = router.get("/reserved/admin");
+    reservedAdminRoute.handler(this::handleGetReservedAdmin);
+  }
+
+  private void registerGetDone(Router router) {
+    Route doneRoute = router.get("/done");
```

Comment:
we should probably add a /admin suffix to this to make it consistent with the reserved admin route. We may want a done route for regular users in the future